### DISCUSSION TITLE
Always encounter The Edge Functions build system is currently busy

### DISCUSSION THREAD

**Discussion by manjieqi** [vercel/community#2278] [Help]
### Summary

17:12:30.553 Error:The Edge Functions build system is currently busy.Please try again later.
17:12:31.447 Deployment completed
17:12:31.052 failed_to_acquire_worker_upload_semaphore:The Edge Functions build system is currently busy.Please try again later.

Manual deployment can be successful, is it because the account is restricted?
I even deleted the account and re-registered, the same code base, it seems that other people can deploy normally.
Usage shows that I don't use many resources.

Does anyone know a solution, it is greatly appreciated.

**Comment by amyegan** [comment]
Thanks for reporting this. I'm looking into it and will let you know when I learn more

**Reply by Henvy-Mango** [reply]
> Thanks for reporting this. I'm looking into it and will let you know when I learn more感谢您的报告。我正在调查，有了更多消息会告诉你的

Hey ! I am also in the same situation.

**Comment by lwabish** [comment]
same here
